src/test.js

```diff
@@ -295,4 +299,20 @@ describe('jsonFetch', async function () {
       expect(response.body).to.equal(undefined);
     });
   });
+
+  describe('logging sensitive information', async function () {
```

Comment:
Maybe there is a better word than "logging" since json-fetch doesn't log anything? Maybe the describe block could just be "error.request data"?